prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,32 @@
+const number1 = 2;
+const number2 = 4;
+let product = number1 * number2;
+console.log('value of product:', product);
+console.log('type of product:', typeof product);
+const charge = 27;
+const payment = 4.5;
+let amountRemaining = charge - payment;
+console.log('value of amountRemaining:', amountRemaining);
+console.log('type of amountRemaining:', typeof amountRemaining);
+const tests = 85;
+const assignments = 79;
+const final = 76;
+let grade = (tests + assignments + final) / 3;
+console.log('value of grade:', grade);
+console.log('type of grade:', typeof grade);
+const firstName = 'Devin';
+const lastName = ' Olesen';
+let fullName = 'Devin' + ' Olesen';
+console.log('value of fullName:', fullName);
+console.log('type of fullName:', typeof fullName);
+const pH = 8;
+let isAcidic = pH < 7;
+console.log('value of isAcidic:', isAcidic);
+console.log('type of isAcidic:', typeof isAcidic);
+const numberOfSoldiers = 301;
+let isSparta = numberOfSoldiers === 300;
+console.log('value of isSparta:', isSparta);
+console.log('type of isSparta:', typeof isSparta);
+let nickname = fullName + ' is the GOAT';
```

Comment:
Close! The exercise asks that you use the "assignment operator"

```
let nickname = fullName;
nickname += ' is the GOAT';
```

Reply:
I have updated the file.